tsconfig.base.json

```
@@ -14,7 +14,13 @@
     "target": "ESNext",
     "types": ["jest", "jest-axe", "jest-puppeteer", "node"],
     "paths": {
-      "govuk-frontend": ["./packages/govuk-frontend/src/govuk/all.mjs"]
+      "govuk-frontend": ["./packages/govuk-frontend/src/govuk/all.mjs"],
```

This is great 🙌

Had a good check and probably need a few more with `/*` wildcards?

```suggestion
      "govuk-frontend": ["./packages/govuk-frontend/src/govuk/all.mjs"],
      "govuk-frontend-stats": ["./shared/stats/src/index.mjs"],
      "govuk-frontend-config": ["./shared/config/index.js"],
      "govuk-frontend-helpers": ["./shared/helpers/index.js"],
      "govuk-frontend-helpers/*": ["./shared/helpers/*.js"],
      "govuk-frontend-helpers/jest/*": ["./shared/helpers/jest/*"],
      "govuk-frontend-lib": ["./shared/lib/index.js"],
      "govuk-frontend-lib/*": ["./shared/lib/*.js"]
```

---

We'll finally be able to remove all the places we include `govuk-frontend`

```json
  "include": ["**/*.js", "**/*.mjs", "../../packages/govuk-frontend"],
  "exclude": ["../../packages/govuk-frontend/dist/**"]
```

Leaving it like:

```json
  "include": ["**/*.js", "**/*.mjs"]
```

---

Or where we have packages that include others:

```json
  "include": [
    "**/*.js",
    "**/*.mjs",
    "../config",
    "../lib",
    "../../packages/govuk-frontend"
  ],
  "exclude": ["./dist/**", "../../packages/govuk-frontend/dist/**"]
```

Won't need to include `"../config"` or `"../lib"` etc

```json
  "include": ["**/*.js", "**/*.mjs"],
  "exclude": ["./dist/**"]
```

Since we'll have them in **tsconfig.base.json** instead 😊